static/.htaccess

```diff
@@ -27,6 +31,18 @@ RewriteRule ^docs/self-managed/operate-deployment/authentication/(.*)$ /docs/sel
 RewriteRule ^docs/self-managed/identity/troubleshooting/?$ /docs/self-managed/identity/troubleshooting/troubleshoot-identity/ [R=301,L]
 RewriteRule ^docs/self-managed/identity/getting-started/?$ /docs/self-managed/identity/getting-started/install-identity/ [R=301,L]
 
+RewriteRule ^docs/1.3/components/overview/(.*)$ /docs/1.3/components/components-overview/$1 [R=301,L]
+RewriteRule ^docs/1.3/components/overview$ /docs/1.3/components/components-overview/ [R=301,L]
+RewriteRule ^docs/1.2/components/overview/(.*)$ /docs/1.2/components/components-overview/$1 [R=301,L]
+RewriteRule ^docs/1.2/components/overview$ /docs/1.2/components/components-overview/ [R=301,L]
+RewriteRule ^docs/1.1/components/overview/(.*)$ /docs/1.1/components/components-overview/$1 [R=301,L]
+RewriteRule ^docs/1.1/components/overview$ /docs/1.1/components/components-overview/ [R=301,L]
+RewriteRule ^docs/1.0/components/overview/(.*)$ /docs/1.0/components/components-overview/$1 [R=301,L]
+RewriteRule ^docs/1.0/components/overview$ /docs/1.0/components/components-overview/ [R=301,L]
+RewriteRule ^docs/0.26/components/overview/(.*)$ /docs/0.26/components/components-overview/$1 [R=301,L]
+RewriteRule ^docs/0.26/components/overview$ /docs/0.26/components/components-overview/ [R=301,L]
+RewriteRule ^docs/0.25/components/overview/(.*)$ /docs/0.25/components/components-overview/$1 [R=301,L]
+RewriteRule ^docs/0.25/components/overview$ /docs/0.25/components/components-overview/ [R=301,L]
```

Comment:
Based on your thoughts here @pepopowitz, you can PR the rewrite rules out. We don't need to create more tech debt if we can avoid it.

I think I've typed 100 different lengthy responses, which can all be similarly boiled down to this - I was using `slug` to avoid some weird URL things happening for our sections and wanted to create nice landing pages with clean URLs. You'll see this on Guides, Components, and Reference (but not APIs & Clients or Self-Managed which are newer sections, where I likely forgot to add a `slug` or realized it was creating other issues).

Likely a better solution is to allow people to navigate to any of those top nav level sections by nice, clean URLs with a simple redirect without slugging. Feel free to create an issue and pop it in the backlog.